pkg/expect/expect.go

```diff
@@ -114,6 +114,11 @@ func (ep *ExpectProcess) Close() error {
 	return err
 }
 
+func (ep *ExpectProcess) Send(command string) error {
```

Comment:
`SendLine` should call `Send`. Maybe get rid of sendline entirely to keep `expect` small?